packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -3045,6 +3045,14 @@
       }
     }
   },
+  "scheduled_report": {
+    "dynamic": false,
+    "properties": {
+      "createdBy": {
+        "type": "keyword"
```

Comment:
we use this to filter scheduled reports by the username of the user who scheduled it.